### PR TITLE
Database mile 3

### PR DESCRIPTION
**Pull request by jasonfelice**
- [ ] Created a table named `owners`
- [ ] Create a table named `species`
- [ ] Added columns `species_id` & `owner_id` which is foreign key referencing their respective tables
- [ ] Inserted data into the `owners` & `species` tables
- [ ] Wrote the following queries: 
one
two
three
four
five
six
seven

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -12,4 +12,16 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
...
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) Values
+    ('Pokemon'),
+    ('Digimon');
```
Comment (keneogu) [inline data.sql]: Add these to your data.sql file
animals table should be modified to meet these requirements
1. if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead.
2. all animals got assigned owners

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @jasonfelice,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (jasonfelice) [reply data.sql]: Thanks!

**Review by elyor-doniyorov — APPROVED** [review]
Hi @jasonfelice, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Made requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**